skeleton/styles/prosilver/template/mcp_demo_body.html.twig

```diff
@@ -1,8 +1,8 @@
 {{ '{' }}% include 'mcp_header.html' %}
 
-<form id="mcp" method="post" action="{{ '{{' }} U_POST_ACTION }}">
+<form id="{{ EXTENSION.vendor_name|lower }}_{{ EXTENSION.extension_name|lower }}" name="{{ EXTENSION.vendor_name|lower }}_{{ EXTENSION.extension_name|lower }}" method="post" action="{{ '{{' }} U_POST_ACTION }}">
```

Comment:
let's keep `_mcp` in the name and id as a suffix.

Reply:
For consistency, you added `_acp` and `_ucp` to the form keys, but missed doing the same for `_mcp`.